test/messages/chainlock.js

```diff
@@ -0,0 +1,174 @@
+    "quorumPublicKey": "03a3fbbe99d80a9be8fc59fd4fe43dfbeba9119b688e97493664716cdf15ae47fad70fea7cb93f20fba10d689f9e3c02",
+    "quorumVvecHash": "bede6b120304eb31d173678bb54ffcb0ab91f8d72d5af78b5047f76e393a26a2",
+    "quorumSig": "9944c544e03a478b401b65cabbb24338872613f7d58ff13ab038ab86418ec70ef1734ff43e965ccb83e02da83b10d44c0f23c630752cfb29b402149a1fc3fad0760e6341a4a1031efad2983c8637d2a461e9bcaf935b7a4dfa225ed2f7771c75",
+    "membersSig": "92eda5c13583577719bea9337b4b9b6286ac11a072de0955b0dc5a012280bb557a53f9643cee7730dabe2d3a4a19042813ef5d39ae92d0015554954011c1e12bc688d4d7672ac33c4001e0dedbfe5d0316f2ad23206d478964ca62d75f50e4d0"
+  };
+
+  var quorum = new QuorumEntry(quorumEntryJSON);
+
+  var expectedRequestId3 = "f79d7cee1eea5839d91da7921920f19258e08b51c7cda01086e52d1b1d86510c";
+
+
+  it('should have clsig a constant', function () {
+    expect(ChainLock.CLSIG_REQUESTID_PREFIX).to.deep.equal('clsig');
+  });
+  describe('instantiation', function () {
+    describe('fromBuffer', function () {
+      it('should be able to parse data from a buffer', function () {
+        var chainLock = ChainLock.fromBuffer(buf2);
+        var chainLockStr = chainLock.toString();
+        expect(chainLockStr).to.be.deep.equal(str2)
+        var chainLockJSON = chainLock.toObject();
+        expect(chainLockJSON).to.be.deep.equal(object2)
+      });
+    });
+
+    describe('fromObject', function () {
+      it('Should be able to parse data from an object', function () {
+        var chainLock = ChainLock.fromObject(object2);
+        var chainLockStr = chainLock.toString();
+        expect(chainLockStr).to.be.deep.equal(str2)
+      });
+    });
+
+    describe('fromString', function () {
+      it('Should be able to parse data from a hex string', function () {
+        var chainLock = ChainLock.fromHex(str2);
+        var chainLockJSON = chainLock.toObject();
+        var chainLockBuffer = chainLock.toBuffer().toString('hex');
+        expect(chainLockJSON).to.be.deep.equal(object2)
+
+        expect(chainLockBuffer).to.be.deep.equal(buf2.toString('hex'))
+      });
+    });
+
+    describe('clone itself', function () {
+      it('can be instantiated from another chainlock', function () {
+        var chainLock = ChainLock.fromBuffer(buf2);
+        var chainLock2 = new ChainLock(chainLock);
+        chainLock2.toString().should.equal(chainLock.toString());
+      });
+    })
+  });
+
+  describe('validation', function () {
+    describe('#verifySignatureAgainstQuorum', function () {
+      it('should verify signature', async function () {
+        var chainLock = new ChainLock(buf2);
+        var isValid = await chainLock.verifySignatureAgainstQuorum(quorum);
+        expect(isValid).to.equal(false);
+
+        //FIXME: Neither DashJ nor DashSync or even dashd have any test vectors of successful verification
+        // from chainlock hex and quorum. We don't yet have ability to unit test successful verify.
+        // TODO: expect().to.equal(true).
```

Comment:
What's the exact issue here? Can we just take a live chainlock and perform verification?

Reply:
Devnet is dead (I can't connect, seems it's because I'm v16), testnet has no chainlocks. 
If you are able to fill that part, that would help me a lot.

Reply:
Alright, let's leave it as it is. We need to create a separate Jira task to reintroduce this case once it's live